src/libYARP_cv/include/yarp/cv/Cv.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2006-2018 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * BSD-3-Clause license. See the accompanying LICENSE file for details.
+ */
+
+#ifndef YARP_CV_CV_H
+#define YARP_CV_CV_H
+
+#include <yarp/sig/Image.h>
+
+#include <type_traits>
+#include <opencv/cv.h>
+#include <opencv2/imgproc/imgproc.hpp>
```

Comment:
I though all the headers in `opencv2` were actually deprecated. Is this inclusion necessary?

Reply:
What do you mean with deprecated? They are not 😄 
What would be more clean is to `#include <opencv2/imgproc.hpp>` 👍

Reply:
Ok, I probably misunderstood some docs!